addons/apecloud-mysql/templates/_helpers.tpl

```diff
@@ -188,7 +188,11 @@ exporter:
   containerName: mysql-exporter
   scrapePath: /metrics
   scrapePort: http-metrics
-
+serviceRefDeclarations:
```

Comment:
Does mysql component depend on etcd?

Reply:
vttablet in the MySQL component also depends on etcd.